Exiled.API/Features/Player.cs

```diff
@@ -845,6 +835,11 @@ public int Scp330Usages
         [Obsolete("Removed from the base-game.", true)]
         public bool HasHands => false;
 
+        /// <summary>
+        /// Gets a dictionary for storing player objects of connected but not yet verified players.
+        /// </summary>
+        internal static ConditionalWeakTable<ReferenceHub, Player> UnverifiedPlayers { get; } = new ConditionalWeakTable<ReferenceHub, Player>();
```

Comment:
Set an initial capacity as we do with the dictionaries above

Reply:
There is no initial capacity argument for a ConditionalWeakTable